### ISSUE TITLE
Extract public and restricted status from Purl for geo objects

### ISSUE THREAD

**Issue by aalsum** [sul-dlss/sul-embed#275]
Public druid PURL xml has:

```
<rightsMetadata>
<access type="read">
<machine>
    <world/>
</machine>
</access>
</code>
```

Restricted druid  PURL xml has:
```
<rightsMetadata>
<access type="read">
<machine>
    <group>Stanford</group>
</machine>
</access>```

**Comment by jkeck** [comment]
Check out https://github.com/sul-dlss/dor-rights-auth

Not sure if this kind of work has already been done there or not.

**Comment by aalsum** [comment]
This issue is already closed by #277  pull request
